engine/player/unique_gear_shadowlands.cpp

```diff
@@ -1984,6 +1984,24 @@ void murmurs_in_the_dark( special_effect_t& effect )
   new dbc_proc_callback_t( effect.player, effect );
 }
 
+// id=336219 driver
+// id=336236 unknown use, triggered by driver
+// id=336234 damage spell
+// id=336222 melee damage spell?
+void dueling_form( special_effect_t& effect )
+{
+  struct dueling_form_t : public generic_proc_t
+  {
+    dueling_form_t( const special_effect_t& e ) : generic_proc_t( e, "dueling_form", 336236 )
```

Comment:
Generally, you want to use the spell id of the damage spell here, so that simc correctly picks up spell attributes, spell school, etc

Reply:
Wowhead says [Dueling Form #336236](https://www.wowhead.com/spell=336236/dueling-form) uses [Duelist's Shot](https://www.wowhead.com/spell=336234/duelists-shot), however Details seems to display [Duelist's Strike](https://www.wowhead.com/spell=336222/duelists-strike). How do I figure out which one to pick?

Reply:
Trust the in-game spell id. If you want to be extra sure (in case Details is doing something weird), you can check the spell id in logs produced by `/combatlog`. In-game combat log is also usable if you have some addon that lets you grab ids from spell links (such as idTip).

Reply:
After some digging I found [a log](https://www.warcraftlogs.com/reports/3Pg2JLzp6Qq9jnXm#fight=3&type=damage-done&source=21&view=events) that uses the trinket. The spell ID is [Duelist's Shot](https://www.wowhead.com/spell=336234/duelists-shot), so I guess it's correct like this.

Reply:
Good find, although you should still use 336234 instead of 336236 for the damage spell. Using the exact same spell ID that shows up in a combat log is important for many reasons. A few of the more visible issues that would come up with 336236 would be that the travel time of the projectile would be instant and the tooltip information will be missing in the HTML report.

I also suppose it's possible that this trinket has two separate procs, one for melee and one for ranged, but since they do the same damage I don't think it should matter much which is used. The only difference between Duelist's Strike (id=336222) and Duelist's Shot (id=336234) are the range and velocity fields.

Reply:
Changed the spell ID to 336234